src/GT.Core/Services/Interfaces/IGTListingService.cs

```diff
@@ -1,19 +1,47 @@
 ﻿using GT.Core.DTO.Impl;
 using GT.Core.FilterModels.Interfaces;
+using GT.Data.Data.GTAppDb.Entities;
 
 namespace GT.Core.Services.Interfaces
 {
 	public interface IGTListingService : IGTService
 	{
 		/// <summary>
-		/// Gets partial listings for list view in UI.
+		/// Gets a list of listings meant for list view in UI, where some properties have been stripped from the view model.
```

Comment:
Propose change to: Get high level view of all relevant listings with possibly applied filter for display

Reply:
Fix in commit.